### PR TITLE
Add TTL description in the required files

### PR DESCRIPTION
**Pull request by apurvabhide17**
Version(s):
OCP 4.13 → OCP 4.16

Issue: [OADP-1794](https://issues.redhat.com/browse/OADP-1794)

Link to docs preview:

- [OADP Creating backup CR](https://80001--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.html)
- [Scheduling backups using Schedule CR](https://80001--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.html)
- [Backing up persistent volumes with CSI snapshots ](https://80001--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-backup-restore-csi-snapshots.html)

QE review:
- [x] QE has approved this change.

This PR will also fix indentation issue for [4.6.6. Scheduling backups using Schedule CR](https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html-single/backup_and_restore/index#oadp-scheduling-backups-doc)

### CODE REVIEW

**Comment by apurvabhide17** [comment]
Thanks @tmalove

**Review by PrasadJoshi12 — APPROVED** [review]

**Comment by apurvabhide17** [comment]
Thanks @PrasadJoshi12

**Comment by apurvabhide17** [comment]
Hello @tmalove, 
Could you please help me to understand why **ci/prow/validate-portal** is failing?  Thanks in advance!

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc [hunk 1]
```diff
@@ -73,10 +72,11 @@ To schedule a backup at specific intervals, enter the `<duration_in_minutes>` in
 ----
 Enter the minutes value between quotation marks (`" "`).
 ====
-
++
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc:75]: try removing this indent, which looks like it is causing the build to start reading from <2>

Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc]: fixed, thanks :)

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc [hunk 2]
```diff
@@ -77,8 +77,9 @@ spec:
 <2> Optional: Specify an array of resources to include in the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified. If unspecified, all resources are included.
 <3> Optional: Specify an array of resources to exclude from the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified.
 <4> Specify the name of the `backupStorageLocations` CR.
-<5> Map of {key,value} pairs of backup resources that have *all* the specified labels.
-<6> Map of {key,value} pairs of backup resources that have *one or more* of the specified labels.
+<5> The `ttl` field defines the retention time of the created backup and the backed up data. For example, if you are using Restic as the backup tool, the backed up data items and data contents of the persistent volumes (PVs) is stored until the backup expires. But storing this data consumes more space in the target backup locations. An additional storage will be consumed with frequent backups, which are created even before other in-progress backups might have timed out.
```
Comment (weshayutin) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc:80]: "even before other in-progress backups might have timed out."  using the term "in-progress" for a backup that has completed but has not yet expired it's TTL is confusing.  "in-progess" backups typically refer to a backup that is currently executing the actual backup.

Unless someone has a better way to describe this state, I would say something like "unexpired completed backups"

This comment holds true for the other sections of the doc as well where "in-progress" is used to describe completed backups that have not expired their ttl. ( time to live )

**Review by weshayutin — CHANGES_REQUESTED** [review]
Please update the use of "in-progress backups"

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc (hunk 2 above)
Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc]: thanks, I've added that

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc [hunk 3]
```diff
@@ -77,8 +77,9 @@ spec:
 <2> Optional: Specify an array of resources to include in the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified. If unspecified, all resources are included.
 <3> Optional: Specify an array of resources to exclude from the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified.
 <4> Specify the name of the `backupStorageLocations` CR.
-<5> Map of {key,value} pairs of backup resources that have *all* the specified labels.
-<6> Map of {key,value} pairs of backup resources that have *one or more* of the specified labels.
+<5> The `ttl` field defines the retention time of the created backup and the backed up data. For example, if you are using Restic as the backup tool, the backed up data items and data contents of the persistent volumes (PVs) is stored until the backup expires. But storing this data consumes more space in the target backup locations. An additional storage will be consumed with frequent backups, which are created even before other unexpired completed backups might have timed out.
```
Comment (ShaunaDiaz) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc:80]:
```suggestion
<5> The `ttl` field defines the retention time of the created backup and the backed up data. For example, if you are using Restic as the backup tool, the backed up data items and data contents of the persistent volumes (PVs) are stored until the backup expires. But storing this data consumes more space in the target backup locations. An additional storage is consumed with frequent backups, which are created even before other unexpired completed backups might have timed out.
```
couple nits

**Review by ShaunaDiaz — COMMENTED** [review]
two nits; verb-subject agreement and future tense

**Comment by apurvabhide17** [comment]
Closing this PR due to conflicts after changing the base branch. I've created a new PR for this - https://github.com/openshift/openshift-docs/pull/83801

**Pull request closed without merging**